config/packages/dev/debug.yaml

```diff
@@ -0,0 +1,4 @@
+debug:
```

Comment:
We can remove this config as we removed the related bundle.

Reply:
fixed